wpilibc/athena/src/Task.cpp

```diff
@@ -51,7 +51,7 @@ std::thread::native_handle_type Task::native_handle() {
  * @return true on success.
  */
 bool Task::Verify() {
-  TASK id = (TASK)m_thread.native_handle();
+  TASK id = reinterpret_cast<TASK>(m_thread.native_handle());
```

Comment:
Yikes!  I don't think this works (the original code that is).  TASK is a `pthread_t*`, but std::thread::native_handle_type (what native_handle() returns) is a `pthread_t`.